README.md

```diff
@@ -69,7 +69,9 @@ Install `marshmallow_dataclass<6.0` if you need marshmallow 2 compatibility.
 
 ## Usage
 
-Use the [`class_schema`](https://lovasoa.github.io/marshmallow_dataclass/html/marshmallow_dataclass.html#marshmallow_dataclass.class_schema) function to generate a marshmallow [Schema](https://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.Schema) class from a [`dataclass`](https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass).
+Use the [`class_schema`](https://lovasoa.github.io/marshmallow_dataclass/html/marshmallow_dataclass.html#marshmallow_dataclass.class_schema)
+function to generate a marshmallow [Schema](https://marshmallow.readthedocs.io/en/latest/api_reference.html#marshmallow.Schema)
```

Comment:
Woops, I've been experimenting with [sentence-per-line-break](https://twitter.com/garybernhardt/status/1167123539834523648?s=20), so that's why all the long lines. Happy with whatever though.